{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-30T20:13:00.000+0000",
      "latitude": 51.58498699932277,
      "estimatedRestoration": "2022-04-30T23:20:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JS5064'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JS5064",
      "reference": "JS5064",
      "name": "SN2 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN2 1AH",
        "SN2 1SJ",
        "SN2 5AA",
        "SN25 3AW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.7769041277246942,
          51.58498699932277
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-30T21:30:00.000+0000",
      "UUID": "6dafe002-08d6-4361-81d1-c7950d885956",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7769041277246942,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-30T19:23:00.000+0000",
      "latitude": 57.69778032006212,
      "estimatedRestoration": "2022-04-30T22:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JS5055'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JS5055",
      "reference": "JS5055",
      "name": "IV30 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "IV30 4JX",
        "IV30 5QS",
        "IV30 5QU",
        "IV30 5QX",
        "IV30 5QY",
        "IV30 5QZ",
        "IV30 5RF",
        "IV30 8RS",
        "IV31 6SP"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -3.356684433354559,
          57.69778032006212
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-30T19:35:00.000+0000",
      "UUID": "4ae0f27b-fa4b-4eea-ba8b-8df1ddafa846",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -3.356684433354559,
      "resolved": false
    }
  ]
}
